core/src/test/java/org/apache/brooklyn/location/byon/ByonLocationResolverTest.java

```diff
@@ -345,7 +345,7 @@ public void testWindowsMachines(String osFamily) throws Exception {
         String spec = "byon";
         Map<String, ?> flags = ImmutableMap.of(
                 "hosts", ImmutableList.of("1.1.1.1", "2.2.2.2"),
-                "osfamily", osFamily
+                "osFamily", "windows"
```

Comment:
Please use the `osFamily` parameter that is being passed in - i.e. `"osFamily", osFamily`